roles/basic/tasks/basic_install_ubuntu.yml

```diff
@@ -1,52 +1,29 @@
----
-# basic installation
-  - name: Update cache - download package information
-    apt:
-      update_cache: yes
-      
-  - name: install packages Ubuntu
-    ansible.builtin.package:
-      name:
-        - aria2
-        - bind9-dnsutils
-        - git 
-        - gzip 
-        - iptables 
-        - bsd-mailx
-        - nmap
-        - sysstat 
-        - tcpdump
-        - tmux 
-        - unzip 
-        - vim 
-        - openssh-server
-      state: latest
-
-  - name: Remove fwupd
-    apt:
-      name: fwupd
-      state: absent
-
-  - name: install python3
-    apt:
-      name: ['python3', 'python3-pip']
-      state: latest
-
-  - name: upgrade all packages
-    apt:
-      name: '*'
-      state: latest
-
-  - name: kernel.panic = 20, reboot 20 seconds after kernel panic
-    lineinfile:
-      path: /etc/sysctl.d/99-sysctl.conf
-      line: "kernel.panic = 20"
-    notify: reload sysctl
-
-  - name: kernel.pid_max = 100000, allow 100.000 Processes at the system
-    lineinfile:
-      path: /etc/sysctl.d/99-sysctl.conf
-      line: "kernel.pid_max = 100000"
-    notify: reload sysctl
-    
-      
+---
+# basic installation
+  - name: Remove fwupd
+    apt:
+      name: fwupd
+      state: absent
+
+  - name: Update apt package list
+    apt:
+      update_cache: true
+      upgrade: dist
+      autoremove: true
+
+  - name: Install python3
+    apt:
+      name: ['python3', 'python3-pip']
+      state: latest
+
+  - name: Kernel.panic = 20, reboot 20 seconds after kernel panic
+    lineinfile:

```

Comment:
We should use the Ansible sysctl module here https://docs.ansible.com/ansible/latest/collections/ansible/posix/sysctl_module.html